### DISCUSSION TITLE
About License

### DISCUSSION THREAD

**Discussion by Liufour** [Hartrik/sand-game-js#4] [General]
Hello, I am developing a dynamic wallpaper apps for the Android platform. When I was playing your game, I found that it looks amazing. I hope to apply this amazing visual effect to my apps, so I studied your code and transplanted the core part to the Unity engine (I haven't publish it to the Internet yet). Includes using 32-bit int type as elements, using element head and tail for processing and rendering, updating in chunks to improve efficiency, and using two frame buffers to achieve blur effects.
In my apps, the part about sand-game-js only shows the flow effect of sand, just like a scene full of water where some temperature is applied to generate water gas. It is not a game and cannot use input to create, move, or destroy any elements.
Finally, I hope to sell this apps at a certain price.
I would like to know if you agree with my actions, as I have reviewed the contents of your license.

**Comment by Hartrik** [comment]
Hello, thanks for writing. Is it okay for you to put a backlink in the description of your app to sandsaga.com? Saying that you were influenced or something like that…

**Reply by Liufour** [reply]
No problem, I can do that, such as add your backlink or copyright information in the description section or in the apps and say that the development of this app was inspired by your game.

**Reply by Hartrik** [reply]
For example, I think this is fine for me:
```
The development of this app was inspired by a falling-sand game called Sand Saga.
Sand Saga (sandsaga.com); © Patrik Harag; all rights reserved
```

Please make it visible at least where the app will be available for download.

Then I agree with your actions as described above, and I’m looking forward to seeing your apps.

**Reply by Liufour** [reply]
Got it, appreciate your permission. I would put the copyright text in the description section of the app and within the app interface. Finally, I also thank you for looking forward to my app, and I will do my best to make it the best it can be.
